plugins/autocomplete/plugin.js

```diff
@@ -586,7 +587,7 @@
 		 * @readonly
 		 * @property {CKEDITOR.template}
 		 */
-		this.itemTemplate = new CKEDITOR.template( '<li data-id="{id}">{name}</li>' );
+		this.itemTemplate = new CKEDITOR.template( '<li data-id="{id}" tabindex="-1">{name}</li>' );
```

Comment:
I'd rather add `[tabindex]` programatically, via `element.$.tabIndex = -1` in `addItem` method. Requiring user to remember to add this attribute to `itemTemplate` seems unnecessary.